OP-1051 fix grammar and spelling

adrienne3500 opened the pull request:
Commas After Introductory Clauses: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_after_introductions.html

Parallel Structure: 
https://owl.purdue.edu/owl/general_writing/mechanics/parallel_structure.html

Dangling Modifers: 
https://owl.purdue.edu/owl/general_writing/mechanics/dangling_modifiers_and_how_to_correct_them.html

Comma Splices, Run-On Sentences, etc.: https://owl.purdue.edu/owl_exercises/sentence_structure/sentence_structure/run_ons_comma_splices_and_fused_sentences.html

Singular-Plural / Subject-Verb Disagreement: https://owl.purdue.edu/owl/general_writing/grammar/subject_verb_agreement.html

In case usage: 
https://dictionary.cambridge.org/us/grammar/british-grammar/in-case-of

Commit:
corrected typo, and corrected spelling of Attendance

Commit:
fixed some comma splices and run-on sentences

Commit:
fixed some ambiguous phrasing

Commit:
fixed ending punctuation

Commit:
fixed singular-plural disagreement

Commit:
fixed incorrect word form and an idiomatic error

Commit:
corrected sentence that previously had two colons

Commit:
fixed positioning of 'and'

adrienne3500 committed:
corrected full infinitive to be gerund

adrienne3500 committed:
fixed more comma splicings

adrienne3500 committed:
fixed a lengthy sentence

adrienne3500 committed:
fixed multiple sentences so commas appear after introductory clauses

adrienne3500 committed:
corrected another full infinitive to be gerund

adrienne3500 committed:
fixed more sentences so commas appear after introductory clauses

adrienne3500 committed:
fixed even more sentences so commas appear after introductory clauses

adrienne3500 committed:
fixed phrase order

adrienne3500 committed:
added another comma after intro clause, and added commas before multiple occurrences of shown below

adrienne3500 committed:
added missing word

adrienne3500 committed:
added commas after intro clauses referring to examples

adrienne3500 committed:
corrected example below to example above

adrienne3500 committed:
added commas after more intro clauses

adrienne3500 committed:
fixed instance of singular-plural disagreement

adrienne3500 committed:
fixed unnatural phrasing

adrienne3500 committed:
assorted grammar fixes, including adding comma before 'but' when there are two independent clauses

adrienne3500 committed:
fixed grammar in second patient example

adrienne3500 committed:
eliminated a duplicated word, and fixed some unnatural phrasing

adrienne3500 committed:
Added commas after intro clauses beginning with 'By', and added missing word

adrienne3500 committed:
decapitalized a field description for consistency

adrienne3500 committed:
fixed grammar for item description, and made fixes so patients are referred to as 'who', not 'that'

adrienne3500 committed:
deleted duplicate word, added missing commas, etc.

adrienne3500 committed:
fixed patient pronoun to 'whom', not 'that'. also rephrased some sentences to eliminate trailing prepositions

adrienne3500 committed:
added more missing commas

adrienne3500 committed:
simplified a phrase, corrected placement of comma

adrienne3500 committed:
edited sentence for parallel structure and verb form agreement

adrienne3500 committed:
fixed a preposition, added a hyphen

adrienne3500 committed:
added comma for clarity

adrienne3500 committed:
re-added a word

adrienne3500 committed:
reverted addition of a word, fixed lengthy phrasing

adrienne3500 committed:
fixed sentence to not include two semicolons. other minor changes for clarity

adrienne3500 committed:
improved phrasing

adrienne3500 committed:
changed 'in the case the' to 'in the event that'

adrienne3500 committed:
improved more phrasing, and added comma after introductory clause

adrienne3500 committed:
improved parallel structure and phrasing in OPD Extended Module section

adrienne3500 committed:
fixed multiple dangling modifiers

adrienne3500 committed:
added commas before 'grouped by' for clarity, and also corrected 'per each' to 'per'

adrienne3500 committed:
added more commas before nonessential elements, as in the previous commit. deleted a nonessential word

adrienne3500 committed:
fixed incorrect comma placement, and fixed more dangling modifiers

adrienne3500 committed:
improved parallel structure, fixed some sentence fragments

dbmalkovsky reviewed (approved):
LGTM although I don't agree with all the changes.   For example, some amount of effort was used in removing "you" from the text (there were over 400+ occurrences).   This PR adds some of those back.   But it is not worth arguing over each one of them.

mwithi commented:
> LGTM although I don't agree with all the changes. For example, some amount of effort was used in removing "you" from the text (there were over 400+ occurrences). This PR adds some of those back. But it is not worth arguing over each one of them.

I agree in removing "you" occurences, @adrienne3500 could you (un)do those please? Thanks a lot!

adrienne3500 committed:
Update UserManual.adoc

Eliminated some occurrences of "you" by either substituting with "one", or by switching to "passive voice"

adrienne3500 committed:
Update UserManual.adoc

pulled from origin, and also added missing comma

adrienne3500 committed:
Update UserManual.adoc

Incorrect usage of "hence"

mwithi reviewed (approved):
LGTM. Thanks a lot!
